### PR TITLE
Bl starting on UI

### PR DESCRIPTION
**Pull request by tluu5**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,2 +1,3 @@
 class ApplicationController < ActionController::Base
+  before_action :authenticate_user!
```
Comment (aldoportillo) [inline app/controllers/application_controller.rb:2]: nice

File: app/views/devise/confirmations/new.html.erb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (aldoportillo) [inline app/views/devise/confirmations/new.html.erb:1]: why do you have this in this commit

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -1,36 +1,23 @@
 <nav class="navbar navbar-expand-lg bg-body-tertiary">
   <div class="container-fluid">
-    <a class="navbar-brand" href="#">Navbar</a>
+    <%= link_to "Photogram", root_path, class: "navbar-brand" %>
```
Comment (aldoportillo) [inline app/views/shared/_navbar.html.erb:3]: nice

File: config/routes.rb [hunk 4]
```diff
@@ -1,9 +1,11 @@
 Rails.application.routes.draw do
+  root "photos#index"
```
Comment (aldoportillo) [inline config/routes.rb:2]: nice

**Review by merrillhuang — COMMENTED** [review]
Counter cache looks good

**Review by merrillhuang — COMMENTED** [review]
Associations look good

**Review by merrillhuang — APPROVED** [review]
Everything looks good.

**Review by aldoportillo — APPROVED** [review]

**Pull request merged**